ClassNotFoundException at start the app

This error is showing on API 16 to 19

```
W/dalvikvm( 3937): threadid=1: thread exiting with uncaught exception (group=0x418d6ce0)
E/AndroidRuntime( 3937): FATAL EXCEPTION: main
E/AndroidRuntime( 3937): Process: org.flyve.admin.dashboard, PID: 3937
E/AndroidRuntime( 3937): java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{org.flyve.admin.dashboard/org.flyve.admin.dashboard.SplashActivity}: java.lang.ClassNotFoundException: Didn't find class "org.flyve.admin.dashboard.SplashActivity" on path: DexPathList[[zip file "/data/app/org.flyve.admin.dashboard-2.apk"],nativeLibraryDirectories=[/data/app-lib/org.flyve.admin.dashboard-2, /vendor/lib, /system/lib]]
E/AndroidRuntime( 3937): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2131)
E/AndroidRuntime( 3937): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2271)
E/AndroidRuntime( 3937): 	at android.app.ActivityThread.access$800(ActivityThread.java:144)
E/AndroidRuntime( 3937): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1205)
E/AndroidRuntime( 3937): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime( 3937): 	at android.os.Looper.loop(Looper.java:136)
E/AndroidRuntime( 3937): 	at android.app.ActivityThread.main(ActivityThread.java:5146)
E/AndroidRuntime( 3937): 	at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime( 3937): 	at java.lang.reflect.Method.invoke(Method.java:515)
E/AndroidRuntime( 3937): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:732)
E/AndroidRuntime( 3937): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:566)
E/AndroidRuntime( 3937): 	at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime( 3937): Caused by: java.lang.ClassNotFoundException: Didn't find class "org.flyve.admin.dashboard.SplashActivity" on path: DexPathList[[zip file "/data/app/org.flyve.admin.dashboard-2.apk"],nativeLibraryDirectories=[/data/app-lib/org.flyve.admin.dashboard-2, /vendor/lib, /system/lib]]
E/AndroidRuntime( 3937): 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
E/AndroidRuntime( 3937): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:497)
E/AndroidRuntime( 3937): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:457)
E/AndroidRuntime( 3937): 	at android.app.Instrumentation.newActivity(Instrumentation.java:1061)
E/AndroidRuntime( 3937): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2122)
E/AndroidRuntime( 3937): 	... 11 more
W/ActivityManager( 2270):   Force finishing activity org.flyve.admin.dashboard/.SplashActivity
W/ActivityManager( 2270): Activity pause timeout for ActivityRecord{41dd2bb0 u0 org.flyve.admin.dashboard/.SplashActivity t2 f}
I/Timeline( 2576): Timeline: Activity_idle id: android.os.BinderProxy@41d37f70 time:48527
I/ActivityManager( 2270): Killing 3532:android.process.acore/u0a4 (adj 15): empty #17
I/Timeline( 2270): Timeline: Activity_windows_visible id: ActivityRecord{41ddc000 u0 com.cyanogenmod.trebuchet/org.cyanogenmod.trebuchet.CustomHomeLauncher t1} time:48942
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -37,18 +37,20 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:recyclerview-v7:27.0.1'
+    testCompile 'junit:junit:4.12'
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
     androidTestCompile('tools.fastlane:screengrab:1.1.0', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
+    compile 'com.android.support:recyclerview-v7:27.0.1'
     compile 'com.android.support:appcompat-v7:27.0.1'
     compile 'com.android.support:design:27.0.1'
     compile 'com.android.support:support-v4:27.0.1'
     compile 'com.orhanobut:logger:2.1.1'
-    testCompile 'junit:junit:4.12'
+    compile 'com.android.support:multidex:1.0.2'
+
 }
 
 import java.util.regex.Pattern

--- a/app/src/main/java/org/flyve/admin/dashboard/DashboardAgent.java
+++ b/app/src/main/java/org/flyve/admin/dashboard/DashboardAgent.java
@@ -28,12 +28,16 @@
 package org.flyve.admin.dashboard;
 
 import android.app.Application;
+import android.content.Context;
 import android.content.pm.ApplicationInfo;
+import android.support.multidex.MultiDex;
 import android.util.Log;
+
 import com.orhanobut.logger.AndroidLogAdapter;
 import com.orhanobut.logger.FormatStrategy;
 import com.orhanobut.logger.Logger;
 import com.orhanobut.logger.PrettyFormatStrategy;
+
 import org.flyve.admin.dashboard.data.DataStorage;
 
 /**
@@ -44,6 +48,11 @@ public class DashboardAgent extends Application {
     private DataStorage cache;
     private static DashboardAgent instance;
     private static Boolean isDebuggable;
+
+    protected void attachBaseContext(Context base) {
+        super.attachBaseContext(base);
+        MultiDex.install(this);
+    }
 
     /**
      * Called when the application is starting before any activity, service or receiver objects have been created

